src/libcollections/string.rs

```diff
@@ -527,9 +595,15 @@ impl String {
         self.vec.capacity()
     }
 
-    /// Reserves capacity for at least `additional` more bytes to be inserted
-    /// in the given `String`. The collection may reserve more space to avoid
-    /// frequent reallocations.
+    /// Increases this `String`'s capacity by at least `additional` bytes.
```

Comment:
"Ensures that the capacity is at least `additional` bytes larger than the length", maybe?